src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/_help.py

```diff
@@ -10,4 +10,19 @@
 helps['appservice'] = """
     type: group
     short-summary: commands for managing your Azure web apps, app service plans, etc
+"""
+
+helps['appservice plan'] = """
+    type: group
+    short-summary: commands for managing your app service plans
+"""
+
+helps['appservice plan update'] = """
+    type: command
+    short-summary: update plan, including 'scale up' and 'scale out'
```

Comment:
Some more text would be helpful here to clarify what you mean by scale up/down (changing the SKU) and scale out/in (changing the number of workers). If that help is at the parameter level, then it's fine if it's not here.